static/js/modules/election-summary.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/* global require, module */
+
+var $ = require('jquery');
+var URI = require('URIjs');
+
+var helpers = require('./helpers');
+
+function ElectionSummary(selector, opts) {
+  this.$elm = $(selector);
+  this.opts = opts;
+
+  this.$count = this.$elm.find('.count');
+  this.$receipts = this.$elm.find('.receipts');
+  this.$disbursements = this.$elm.find('.disbursements');
+
+  this.fetch();
+  this.$elm.find('a.button--election').attr('href', this.buildUrl());
```

Comment:
I think the `a` makes sense, since this behavior can only apply to anchor tags. Agreed re. `js-` prefixed classes; will update.